src/1week/06.js

```diff
@@ -1,46 +1,32 @@
-const shopping_cart = [];
-let shopping_cart_total = 0;
+let shopping_cart = [];
+const MINPRICEOFFREESHIPPING  = 20
 
-export function add_item_to_cart(cart, name, price) {
-  const newCart = [
-    ...cart,
-    {
-      name: name,
-      price: price,
-    },
-  ];
-  calc_cart_total();
-}
+// C 
+const tax = (totalPrice) => totalPrice * 0.1;
```

Comment:
```suggestion
const tax = totalPrice => totalPrice * 0.1;
```
화살표에서 param가 하나일 땐 괄호를 빼는게 깔끔한 것 같습니다! lint에도 그런 규칙이 있더라구요